src/WebJobs.Script/Config/HostConfigurationProfile.cs

```diff
@@ -16,23 +15,25 @@ public sealed class HostConfigurationProfile
         // This ensures tests will fail if these values are changed without updating the test also.
         private const string DefaultProfile = "default";
 
-        private const string McpCustomerHandlerProfile = "mcp-custom-handler";
+        private const string McpCustomHandlerProfile = "mcp-custom-handler";
+
+        private const string WebAppCustomHandlerProfile = "web-app-custom-handler";
 
         // Make sure to update this as new profiles are added.
-        private const string SupportedValues = $"'', '{DefaultProfile}', '{McpCustomerHandlerProfile}'";
+        private const string SupportedValues = $"'', '{DefaultProfile}', '{McpCustomHandlerProfile}', '{WebAppCustomHandlerProfile}'";
+
+        private static readonly Dictionary<string, string> CommonHttpCustomHandlerConfiguration = new()
+        {
+            { ConfigurationPath.Combine(ConfigurationSectionNames.CustomHandler, ScriptConstants.EnableProxyingHttpRequest), "true" },
```

Comment:
small nit, I like using the new(er) dictionary init syntax. It is more consistent with regular dictionary syntax


```suggestion
            [ConfigurationPath.Combine(ConfigurationSectionNames.CustomHandler, ScriptConstants.EnableProxyingHttpRequest)] = "true",
```